### PR TITLE
modified out of trigger FCAL channels finding, and swapped row<->col

### PR DESCRIPTION
**Pull request by ilyalarin**
FCAL row columns were swapped - corrected, and made out of trigger channels finding more straightforward, log scale for this histogram was removed

### CODE REVIEW

**Comment by gluex** [comment]
Test status for this pull request: SUCCESS
 
  Summary: [/work/halld/pull_request_test/halld_recon^ilarin_061324/tests/summary.txt](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/tests-summary.txt)
 Logs: [/work/halld/pull_request_test/halld_recon^ilarin_061324/tests/log](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/tests-logs)
 
 Build log: [/work/halld/pull_request_test/halld_recon^ilarin_061324/make_ilarin_061324.log](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/make_ilarin_061324.log)
 Build report: [/work/halld/pull_request_test/halld_recon^ilarin_061324/report_ilarin_061324.txt](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/report_ilarin_061324.txt)
 Location of build: /work/halld/pull_request_test/halld_recon^ilarin_061324

**Comment by gluex** [comment]
Test status for this pull request: SUCCESS
 
  Summary: [/work/halld/pull_request_test/halld_recon^ilarin_061324/tests/summary.txt](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/tests-summary.txt)
 Logs: [/work/halld/pull_request_test/halld_recon^ilarin_061324/tests/log](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/tests-logs)
 
 Build log: [/work/halld/pull_request_test/halld_recon^ilarin_061324/make_ilarin_061324.log](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/make_ilarin_061324.log)
 Build report: [/work/halld/pull_request_test/halld_recon^ilarin_061324/report_ilarin_061324.txt](https://halldweb.jlab.org/pull_request_test/halld_recon^ilarin_061324/report_ilarin_061324.txt)
 Location of build: /work/halld/pull_request_test/halld_recon^ilarin_061324

**Review by aaust — APPROVED** [review]
Look good

File: src/plugins/monitoring/FCAL_cpp/JEventProcessor_FCAL_cpp.cc [hunk 1]
```diff
@@ -201,18 +203,42 @@ jerror_t JEventProcessor_FCAL_cpp::fini(void)
...
-      h_fcal_nclust->SetBinError(i+1, j+1, sqrt(fcal_nclust[i][j])/double(nfcaltriggers));
+      h_fcal_nclust->SetBinContent(i+1, j+1, fcal_nclust[j][i]/double(nfcaltriggers));
+      h_fcal_nclust->SetBinError(i+1, j+1, sqrt(fcal_nclust[j][i])/double(nfcaltriggers));
 
       if(ntoftriggers) {
-        h_fcal_nhitonly->SetBinContent(i+1, j+1,    fcal_nhitonly[i][j] /double(ntoftriggers));
-        h_fcal_nhitonly->SetBinError(i+1, j+1, sqrt(fcal_nhitonly[i][j])/double(ntoftriggers));
+        int n0 = fcal_nhitonly[j][i][0], n1 = fcal_nhitonly[j][i][1];
+
+        double p, ep;
```
[the remark this answers is not in the document]
Comment (aaust) [reply src/plugins/monitoring/FCAL_cpp/JEventProcessor_FCAL_cpp.cc]: Was adressed

**Pull request merged**